datafusion/physical-plan/src/aggregates/mod.rs

```diff
@@ -1025,6 +1031,46 @@ impl ExecutionPlan for AggregateExec {
     fn cardinality_effect(&self) -> CardinalityEffect {
         CardinalityEffect::LowerEqual
     }
+
```

Comment:
Could you add a few test to verify that:

1) this doesn't get pushed down (`a` missing in the second grouping set)
```
SELECT a, b, COUNT(*)
FROM table
WHERE $FILTER(a)
GROUP BY GROUPING SETS ((a, b), (b))
```

2) this gets pushed down:
```
SELECT a, b, COUNT(*)
FROM table
WHERE $FILTER(b)
GROUP BY GROUPING SETS ((a, b), (b))
```
Thanks!